### PR TITLE
BS-223 | Updated libraries versions to fix vulnerability (Critical and High) reported by Trivy

### PR DESCRIPTION
**Pull request by manimaarans**
https://bahmni.atlassian.net/browse/BS-223

Fixing vulnerabilities Critical and High

link to the report before fixing
https://docs.google.com/spreadsheets/d/1aRkfZ-7rop6VoRt2DE_kFrd_CdbxB1rhrldx6VbGUW8/edit#gid=877149650

link to the report after fixing
https://docs.google.com/spreadsheets/d/1aRkfZ-7rop6VoRt2DE_kFrd_CdbxB1rhrldx6VbGUW8/edit#gid=1997548896

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,11 +23,24 @@
...
+
 		<dependency>
 			<groupId>ca.uhn.hapi.fhir</groupId>
 			<artifactId>hapi-fhir-structures-r4</artifactId>
-			<version>6.4.0</version>
+			<version>6.8.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.jetbrains</groupId>
+			<artifactId>annotations</artifactId>
```
Comment (kaicode) [inline pom.xml:41]: Is this library needed? There were no code changes so I assume there are no annotations of this type used in this project?

**Review by kaicode — COMMENTED** [review]
Looks good to me. The build works. I have just left one question about the extra dependency.

File: pom.xml (hunk 1 above)
Comment (manimaarans) [reply pom.xml:41]: Hi @kaicode , We are using it here 
https://github.com/IHTSDO/snomed-fhir-cds-service/blob/develop/src/main/java/org/snomed/cdsservice/service/CDSService.java#L45

from https://github.com/IHTSDO/snomed-fhir-cds-service/blob/develop/src/main/java/org/snomed/cdsservice/service/CDSService.java#L5

Previously this library contained the dependency as part of it 
https://github.com/IHTSDO/snomed-fhir-cds-service/blob/develop/pom.xml#L30

but the updated library doesn't contain it so had to add them in the Pom explicitly

Comment (kaicode) [reply pom.xml:41]: Perfect thanks 👍

**Pull request merged**